src/oc_erchef/rebar.lock

```diff
@@ -67,8 +67,8 @@
   0},
  {<<"goldrush">>,{pkg,<<"goldrush">>,<<"0.1.9">>},1},
  {<<"ibrowse">>,
-  {git,"git://github.com/chef/ibrowse.git",
-       {ref,"1df08a6aca30614f69126451957af6a3d0aeb7b2"}},
+  {git,"git://github.com/cmullaparthi/ibrowse.git",
```

Comment:
Let us keep using the chef fork. It has some fixes made to fix some transient issues seen in the pipelines. We should create a ticket to update our fork.

Reply:
Pointed all references to ibrowse to chef/ibrowse.